src/Calendar/Calendar.js

```diff
@@ -166,6 +167,7 @@ class Calendar extends Component {
 
         this.setState({
             currentDateDisplayed: newDate,
+            screenReaderText: this.props.localizedText.dayInstructions,
```

Comment:
Should this be `monthInstructions` and the one below be `yearInstructions`?

Reply:
This is upon selection, so this is when it switches back to the day view.

Reply:
Ah ok I see it now.